### PR TITLE
For #19923: renew telemetry probes

### PR DESCRIPTION
**Pull request by eliserichards**
For #19923 

~Waiting on confirmation from Vesta~ ✅ (see google doc)

It's probably easier to review these commits separately so you can see the date changes apart from the email changes

- Update probe expiration dates from May, June, and July
   - https://docs.google.com/document/d/1NGlnTa9TPyTnd3ciUPbwujbITjkX8p8vJybXcZrrM2w/edit 
   - We are not including `downloads`, `about_page`, or `contextual_hint.tracking_protection` in this PR. Separate issues are filed for these ([1](https://jira.mozilla.com/browse/FNX-22778), [2](https://jira.mozilla.com/browse/FNX-22779), [3](https://jira.mozilla.com/browse/FNX-22780)) to handle before July expirations.
   - Expiration set for December 10, 2021 so that renewal doesn't interfere with holiday travel.
- Update `notification_emails` to firefox-android-team@mozilla.com
   - Confirmed with @vesta0 and @amedyne 

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by gabrielluong — COMMENTED** [review]
Looks good, I believe @pocmo mentioned about getting too much telemetry spam so perhaps we should ping around and figure out an alternative to `firefox-android-team@mozilla.com` email. Ideally, I think we should just create a new email group for the telemetry owners.

**Comment by eliserichards** [comment]
`firefox-android-team@mozilla.com` is the group change approved by @amedyne and @vesta0

**Comment by eliserichards** [comment]
# Request for Data Collection Renewal

You must receive Data Review from a
[Data Steward](https://wiki.mozilla.org/Data_Collection)
on a filled-out Request before shipping your renewed data collection.**

1) Provide a link to the initial Data Collection Review Request for this collection.
There are a large number of these, sorry in advance:
May: 
- https://github.com/mozilla-mobile/fenix/pull/16727 
- https://github.com/mozilla-mobile/fenix/pull/16739
- https://github.com/mozilla-mobile/fenix/pull/16915 

June: 
- https://github.com/mozilla-mobile/fenix/pull/1896 
- https://github.com/mozilla-mobile/fenix/pull/5704 
- https://github.com/mozilla-mobile/fenix/pull/5886 
- https://github.com/mozilla-mobile/fenix/pull/5975 
- https://github.com/mozilla-mobile/fenix/pull/6352 
- https://github.com/mozilla-mobile/fenix/pull/6601 
- https://github.com/mozilla-mobile/fenix/pull/6746 
- https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877
- https://github.com/mozilla-mobile/fenix/pull/16968
- https://github.com/mozilla-mobile/fenix/pull/17049
- https://github.com/mozilla-mobile/fenix/pull/12114#pullrequestreview-445245341 
- https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877 
- https://github.com/mozilla-mobile/fenix/pull/13494#pullrequestreview-474050499 
- https://github.com/mozilla-mobile/fenix/pull/15605#issuecomment-702365594
- https://github.com/mozilla-mobile/fenix/pull/11940/ 


July: 
https://github.com/mozilla-mobile/fenix/pull/12114#pullrequestreview-445245341 
https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877 
https://github.com/mozilla-mobile/fenix/pull/13494#pullrequestreview-474050499 
https://github.com/mozilla-mobile/fenix/pull/15605#issuecomment-702365594
https://github.com/mozilla-mobile/fenix/pull/11940/ 
https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877
https://github.com/mozilla-mobile/fenix/pull/1067#issuecomment-474598673 
https://github.com/mozilla-mobile/fenix/pull/18143
https://github.com/mozilla-mobile/fenix/pull/1677 
https://github.com/mozilla-mobile/fenix/pull/18143
https://github.com/mozilla-mobile/fenix/pull/1214#issue-264756708 
https://github.com/mozilla-mobile/fenix/pull/5098#issuecomment-529658996 
https://github.com/mozilla-mobile/fenix/pull/6310 
https://github.com/mozilla-mobile/fenix/pull/5090 
https://github.com/mozilla-mobile/fenix/pull/5975 
https://github.com/mozilla-mobile/fenix/pull/11533 
https://github.com/mozilla-mobile/fenix/pull/1214#issue-264756708 
https://github.com/mozilla-mobile/fenix/pull/1344#issuecomment-479285010 
https://github.com/mozilla-mobile/fenix/issues/16076#issuecomment-726216734 
https://github.com/mozilla-mobile/fenix/pull/1708

2) When will this collection now expire?

All probes that have expired in May, June, or July, with the exception of `downloads`, `about_page, and `contextual_hint.tracking_protection`, will not expire in December 2021. These dates are either 12-01-2021 or 12-10-2021 depending on the original add date of the telemetry. See [doc](https://docs.google.com/document/d/1NGlnTa9TPyTnd3ciUPbwujbITjkX8p8vJybXcZrrM2w/edit) for breakdown of individual probes and expiration dates.

3) Why was the initial period of collection insufficient?

We are continuing to collect this data because it tells us basic usage patterns of the app and gives us information about our new features and what is useful/working. This data is being actively used by the product team for MR2 decisions.

**Comment by travis79** [comment]
> # Request for Data Collection Renewal
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        There are a large number of these, sorry in advance:
>        May:
> 
> 
>     * #16727
> 
>     * #16739
> 
>     * #16915
> 
> 
> June:
> 
>     * #1896
> 
>     * #5704
> 
>     * #5886
> 
>     * #5975
> 
>     * #6352
> 
>     * #6601
> 
>     * #6746
> 
>     * [#13958 (comment)](https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877)
> 
>     * #16968
> 
>     * #17049
> 
>     * [#12114 (review)](https://github.com/mozilla-mobile/fenix/pull/12114#pullrequestreview-445245341)
> 
>     * [#13958 (comment)](https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877)
> 
>     * [#13494 (review)](https://github.com/mozilla-mobile/fenix/pull/13494#pullrequestreview-474050499)
> 
>     * [#15605 (comment)](https://github.com/mozilla-mobile/fenix/pull/15605#issuecomment-702365594)
> 
>     * #11940
> 
> 
> July:
> [#12114 (review)](https://github.com/mozilla-mobile/fenix/pull/12114#pullrequestreview-445245341)
> [#13958 (comment)](https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877)
> [#13494 (review)](https://github.com/mozilla-mobile/fenix/pull/13494#pullrequestreview-474050499)
> [#15605 (comment)](https://github.com/mozilla-mobile/fenix/pull/15605#issuecomment-702365594)
> #11940
> [#13958 (comment)](https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877)
> [#1067 (comment)](https://github.com/mozilla-mobile/fenix/pull/1067#issuecomment-474598673)
> #18143
> #1677
> #18143
> [#1214 (comment)](https://github.com/mozilla-mobile/fenix/pull/1214#issue-264756708)
> [#5098 (comment)](https://github.com/mozilla-mobile/fenix/pull/5098#issuecomment-529658996)
> #6310
> #5090
> #5975
> #11533
> [#1214 (comment)](https://github.com/mozilla-mobile/fenix/pull/1214#issue-264756708)
> [#1344 (comment)](https://github.com/mozilla-mobile/fenix/pull/1344#issuecomment-479285010)
> [#16076 (comment)](https://github.com/mozilla-mobile/fenix/issues/16076#issuecomment-726216734)
> #1708
> 
>     1. When will this collection now expire?
> 
> 
> All probes that have expired in May, June, or July, with the exception of `downloads`, `about_page, and `contextual_hint.tracking_protection`, will now expire in December 2021. These dates are either 12-01-2021 or 12-10-2021 depending on the original add date of the telemetry. See [doc](https://docs.google.com/document/d/1NGlnTa9TPyTnd3ciUPbwujbITjkX8p8vJybXcZrrM2w/edit) for breakdown of individual probes and expiration dates.
> 
>     1. Why was the initial period of collection insufficient?
> 
> 
> We are continuing to collect this data because it tells us basic usage patterns of the app and gives us information about our new features and what is useful/working. This data is being actively used by the product team for MR2 decisions.

# Data Collection Renewal Review

1) Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?

Yes

2) Is the data collection covered by the existing Firefox Privacy Notice?

Yes

## Result

data-review+

**Review by gabrielluong — APPROVED** [review]

File: app/metrics.yaml [hunk 1]
```diff
@@ -432,7 +432,7 @@ events:
       - interaction
     notification_emails:
       - fenix-core@mozilla.com
```
Comment (gabrielluong) [inline app/metrics.yaml:434]: You missed this one.

**Pull request merged**